index.html

```diff
@@ -1,39 +1,348 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="stylesheet" href="style.css">
-    <link href='http://fonts.googleapis.com/css?family=Roboto' rel='stylesheet' type='text/css'>
-    <title>Document</title>
-</head>
-<body>
-
-    
-        <nav class="nav">
-            <label class="logo">KAZANEZA GENTIL</label>
-            <a href="#">
-                <img class="navImage" src="./Images/ic_menu.svg" alt="Navigation menu">
-            </a>
-        </nav>
-    
-        <header class="container">
-            <div class="headline"> Hey there. <br/> I'm Kazaneza</div>
-            <div class="title">I’am a Software Developer</div>
-            <div class="introduction"> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, on’t hestiate to contact me.</div>
-
-            <div class="icons">
-                <div class="button twitter"><img  src="./images/Social Media.png" alt=""></div>
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./styles/mobile.css" />
+    <link
+      media="screen and (min-width: 768px)"
+      rel="stylesheet"
+      href="./styles/desktop.css"
+    />
+    <link
+      href="http://fonts.googleapis.com/css?family=Roboto"
+      rel="stylesheet"
+      type="text/css"
+    />
+    <title>Portfolio Kazaneza</title>
+  </head>
+  <body>
+    <header>
+      <nav class="nav">
+        <label class="logo"><a href="">KAZANEZA GENTIL</a></label>
+        <ul class="desktop-nav">
+          <li class="desktop-link"><a href="#">Portfolio</a></li>
+          <li class="desktop-link"><a href="#">About</a></li>
+          <li class="desktop-link"><a href="#">contact</a></li>
+        </ul>
+        <a href="#" class="mobile-nav">
+          <img
+            class="navImage"
+            src="./Images/ic_menu.svg"
+            alt="Navigation menu"
+          />
+        </a>
+      </nav>
```

Comment:
- [ ] Good job :+1:  implementing this website, However your header should stick to the design as it is a crucial part of the project requirement. A design stick to another one if argument such as `font`, `color` `margin` and so on match :100:  Kindly make sure you correct that

Your design

![Screenshot from 2022-12-10 18-19-23](https://user-images.githubusercontent.com/80458199/206864915-bd909a75-ae56-41d7-9e49-137ad511afea.png)

Figma design

![Screenshot from 2022-12-10 18-19-54](https://user-images.githubusercontent.com/80458199/206864919-785d993c-2d5a-4f03-8525-9a4c13fb460c.png)

Reply:
Fixed.